mne_bids/write.py

```diff
@@ -821,6 +887,23 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         inferred from the stim channel using `mne.find_events`.
     event_id : dict | None
         The event id dict used to create a 'trial_type' column in events.tsv
+    anonymize : dict | None
+        If None is provided (default) no anonymization is performed.
+        If a dictionary is passed, data will be anonymized; identifying data
+        structures such as study date and time will be changed.
+        `daysback` is a required argument and `keep_his` is optional, these
+        arguments are passed to :func:`mne.io.anonymize_info`.
+
+        `daysback` : int
+            Number of days to move back the date with a 100 year offset as 0.
+            To keep relative dates for a subject use the same `daysback`.
+            According to BIDS specifications, the number of days back must be
+            great enough that the date is before 1925.
```

Comment:
we did not make progress on the issue on `bids-specification`. Maybe you can help push it forward to change the date?